.github/workflows/iroha2-dev-pr.yml

```diff
@@ -149,6 +143,10 @@ jobs:
           cargo build --bin iroha
       - name: Setup test Iroha 2 environment on the bare metal
         run: |
+        # FIXME: not sure how to install it in the right way. `requirements.txt` triggers
```

Comment:
Maybe create [venv](https://docs.python.org/3/library/venv.html) and install required package into it?

Reply:
Feels like an overkill for CI.. But will probably work fine.

`pip` also has a `--break-system-packages` flag that makes it ignore the fact that you are supposed to install packages via the system package manager & it will probably also work here

Reply:
Added `--break-system-packages`